src/documents/activities/CauseAndEffect.html.md.handlebars

```diff
@@ -4,35 +4,39 @@ layout: content
 category: Activities
 ---
 
-This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is to define a possible chain of events that could take you from a specific cause to an effect.
+This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is
+to define a possible chain of events that could take you from a specific cause to an effect.
 
-Causes defined in this exerise are everyday tasks such as:
+Causes defined in this exercise are everyday tasks such as:
 * share meeting minutes in a form that your colleague using a screen reader can access, or
 * coach an employee in managing a pension.
 
 While the effects might include:
 * grandmother in Ecuador connects with children in LA, or
 * polar bear is saved from ice flow in Alaska.
 
-The effects are as far away from the causes as possible. 
+The effects are as far away from the causes as possible.
 
-Use the accompanying [Cause and Effect card deck](/images/CauseAndEffectCards.pdf) (downloadable pdf), or think up your own examples. These should be as specific and diverse as possible, and the relationship between cause and effect should not immediately be clear.
+Use the accompanying [Cause and Effect card deck](/images/CauseAndEffectCards.pdf) (downloadable pdf), or think up your
```

Comment:
additional change. A PR should have minimum change for fixing an issue.

Reply:
these are the spelling changes